index.js

```diff
@@ -3,7 +3,7 @@
  */
 
 import {AppRegistry} from 'react-native';
```

Comment:
> just tested lint-staged by doing a new commit it works well. You can review

Alright, but theses files are already committed and should also be prettified. You may do that in another PR if you prefer.